examples/neural_compressor/question-answering/README.md

```diff
@@ -56,6 +56,21 @@ python run_qa.py \
     --output_dir /tmp/squad_output
 ```
 
+The following example applies post-training static quantization on a DistilBERT fine-tuned on the SQuAD1.0 dataset and uses INC IPEX backend.
+
+```bash
+python run_qa.py \
+    --model_name_or_path bert-large-uncased-whole-word-masking-finetuned-squad \
+    --dataset_name squad \
+    --apply_quantization \
+    --quantization_approach static \
+    --do_train \
+    --do_eval \
+    --verify_loading \
+    --output_dir /tmp/squad_output \
+    --backend ipex
```

Comment:
Instead of adding an entire command line example, I think it would be great to have a sentence explaning that IPEX can be enabled by setting the backend argument accordingly

Reply:
Edited with a short explanation.